<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="user">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 border-style:solid;  
       background: transparent;  
       background-color: rgb(85, 170, 255); 
         font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  
</string>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>199</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radio1">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);


  
  
  font: 10pt &quot;微软雅黑&quot;;
        
          

color:#000000;</string>
      </property>
      <property name="text">
       <string>财务</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio2">
      <property name="styleSheet">
       <string notr="true">


  
  
  font: 10pt &quot;微软雅黑&quot;;
        
          

color:#000000;</string>
      </property>
      <property name="text">
       <string>管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio3">
      <property name="styleSheet">
       <string notr="true">


  
  
  font: 10pt &quot;微软雅黑&quot;;
        
          

color:#000000;</string>
      </property>
      <property name="text">
       <string>用户</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  font: 12pt &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>32</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/4.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>user</zorder>
  <zorder>password</zorder>
  <zorder>pushButton</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
